app/components/player-card.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import PlayerCard from "./player-card";
+
+const meta: Meta<typeof PlayerCard> = {
+	/* 👇 The title prop is optional.
+	 * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+	 * to learn how to generate automatic titles
+	 */
+	title: "PlayerCard",
+	component: PlayerCard,
+	argTypes: {}
+};
+
+export default meta;
+type Story = StoryObj<typeof PlayerCard>;
+
+export const Card: Story = {
+	args: {}
```

Comment:
I would add the args to the component so the card is not blank when you open storybook, should take 5 minutes to add props

Reply:
Yes - I second this. @SethY8s any useful links?